File: 0191-number-of-1-bits/0191-number-of-1-bits.java
class Solution {
    public int hammingWeight(int n) {
        String bits=Integer.toBinaryString(n);
        int count=0;
        for(int i=0;i<bits.length();i++){
            if(bits.charAt(i)=='1'){
                count++;
            }
        }
        return count;
    }
}

File: 0191-number-of-1-bits/0191-number-of-1-bits.js
/**
 * @param {number} n
 * @return {number}
 */
var hammingWeight = function(n) {
    let bits=n.toString(2);
    let count=0;
    for(let i=0;i<bits.length;i++){
        if(bits.charAt(i)=='1'){
            count++;
        }
    }
    return count;
};